### PR TITLE
Documentation for raspi-config command line interface

### LINKED ISSUE #2454: Add documentation for non-interactive use of raspi-config
Need to document non-interactive use of `raspi-config` through using the `nonint` argument. See RPi-Distro/raspi-config#123 for background details.

### PR DESCRIPTION
**Pull request by aallan**
This PR should close #2454.

### CODE REVIEW

**Comment by aallan** [comment]
@XECDesign could you have a quick look over for technical accuracy? Many thanks.

File: documentation/asciidoc/computers/configuration/raspi-config.adoc [hunk 1]
```diff
@@ -260,3 +262,456 @@ Although it can be run at any time, some of the options may have difficulties if
...
+
+The `raspi-config` tool can also be run in a non-interactive mode, which is useful for setting up a Raspberry Pi image for distribution.
+
+----
+sudo raspi-config nonint <command> <arguments>
+----
+
+The `sudo` is required because you will be changing files that you do not own as the `pi` user.
+
+NOTE: There is no consistent meaning for `0` and `1` in arguments. Each function wll document what `0` and `1` mean for that function.
```
Comment (cinderblock) [inline documentation/asciidoc/computers/configuration/raspi-config.adoc:276]: `wll` -> `will`

Comment (aallan) [reply documentation/asciidoc/computers/configuration/raspi-config.adoc]: Fixed in 0423f13

**Comment by aallan** [comment]
YOLO

**Pull request merged**